### ISSUE TITLE
SyntaxWarning

### ISSUE THREAD

**Issue by jonyscathe** [ragardner/tksheet#205]
Hi,
I'm getting a syntax warning in Python 3.12.0

```
/home/.../.cache/pypoetry/virtualenvs/...-af0V9KsP-py3.12/lib/python3.12/site-packages/tksheet/_tksheet_vars.py:9: SyntaxWarning: invalid escape sequence '\$'
  symbols_set = set("""!#\$%&'()*+,-./:;"@[]^_`{|}~>?= """)
```

Am guessing it should really be  `symbols_set = set("""!#\\$%&'()*+,-./:;"@[]^_`{|}~>?= """)`?

**Comment by rdbende** [comment]
I think it actually should just be a raw string, where the backslash doesn't have any special meaning:

```py
symbols_set = set(r"""!#\$%&'()*+,-./:;"@[]^_`{|}~>?= """)
```

**Comment by ragardner** [comment]
Thanks very much for pointing this out, it should be fixed in tksheet version `6.3.4`
